js/views/inbox_view.js

```diff
@@ -108,6 +108,7 @@
             submitDebugLog          : i18n('submitDebugLog'),
             settings                : i18n('settings'),
             restartSignal           : i18n('restartSignal'),
+            version                 : chrome.runtime.getManifest().version
```

Comment:
no idea really if this is even the right place/view to throw this in
![noidea](https://cloud.githubusercontent.com/assets/174176/14984685/a52fc016-114c-11e6-8d9a-0f5f4b0f584d.jpg)